sass/app.scss

```diff
@@ -0,0 +1,309 @@
+@import "bourbon";
+@import "neat";
+@import "mixins";
+@import "normalize";
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
+@import "font-awesome";
+html {
+  box-sizing: border-box;
+  background: url(img/bg.jpg);
+  font-family: sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+}
+
+.top-nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.top-nav a{
+  color: #A4A09D;
+}
+
+.top-nav-icons .fa {
+  font-size: 40px;
+  margin: 0.3rem;
+}
+
+.top-nav .drop-down {
+  padding-right: 6rem;
+  line-height: 3.0rem;
+  color: rgba(156, 162, 160, 0.63);
+}
+
+.fa-ellipsis-v{
+  color: #C4C4C4;
+}
+
+.top-nav .drop-down i {
+  font-size: 10px;
+  padding-left: 1rem;
+  line-height: inherit;
+}
+
+.red-button {
+  max-width: 1040px;
+  position: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  padding-top: 0.5rem;
+}
+.red-button button {
+  flex: 1 0 calc(25% - 10px);
+  text-align: center;
+  margin: 0.2rem;
+  width: 23%;
+}
+
+.fa-facebook-official {
+  color: #445897;
+}
+
+.fa-rss-square {
+  color: #CB8E31;
+}
+
+.fa-twitter-square{
+  color: #698DC6;
+}
+
+.fa-youtube-square {
+color: #B1372C;
+}
+
+.red-button {
+  color: #DCA7AA;
+}
+
+.red-button button {
+  background-color: #B4292E;
+  border: solid #B4292E;
+}
+
+section .header-pic {
+  max-width: 1040px;
+  position: relative;
+}
+.top-img {
+  width: 100%;
+  border: solid white 10px;
+  height: 500px;
+  align-items: center;
+  margin-top: 0.6rem;
+}
+.top-img-logo {
+  position: absolute;
```

Comment:
Currently this is being positioned based on the body, not the header. You will want to add a `position: relative` to the `.header-pic` so that you compare with that parent.